test/models.jl

```diff
@@ -18,18 +20,20 @@ using Random
                     @testset "Matrix of inputs" begin
                         @test size(logits(model[:model], X))[2] == size(X, 2)
                         @test size(probs(model[:model], X))[2] == size(X, 2)
+                        @test model[:model].likelihood == value[:data].likelihood
                     end
                     @testset "Vector of inputs" begin
                         @test size(logits(model[:model], X[:, 1]), 2) == 1
                         @test size(probs(model[:model], X[:, 1]), 2) == 1
+                        @test model[:model].likelihood == value[:data].likelihood
```

Comment:
a short comment explaining these would be nice

Reply:
Sure, will add this